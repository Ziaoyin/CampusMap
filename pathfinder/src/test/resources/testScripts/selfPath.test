# tests if path to self has a weight of 0.000
CreateGraph graph
AddNode graph n1
FindPath graph n1 n1
AddNode graph n2
AddEdge graph n1 n2 0.0
FindPath graph n1 n1
AddNode graph n3
AddEdge graph n1 n3 1.0
FindPath graph n1 n1