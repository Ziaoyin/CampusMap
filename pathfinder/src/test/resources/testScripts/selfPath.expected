# tests if path to self has a weight of 0.000
created graph graph
added node n1 to graph
path from n1 to n1:
total cost: 0.000
added node n2 to graph
added edge 0.000 from n1 to n2 in graph
path from n1 to n1:
total cost: 0.000
added node n3 to graph
added edge 1.000 from n1 to n3 in graph
path from n1 to n1:
total cost: 0.000